gen_linear_regression/gamma.qmd

```diff
@@ -0,0 +1,24 @@
+---
+title: Gamma model
+---
+
+## Description
+
+Linear regression with gamma distributed residuals.
+
+## Definition
+
+For continuous outcome $y$ lower bounded at zero and predictors $x$, the model is:
+
+$$
+\begin{align}
+y_i \sim \text{gamma}(s, \alpha/exp(\eta)) \\
+\eta_i = \alpha + \beta \cdot x_i
+\end{align}
+$$
```

Comment:
The Gamma distribution can be a tricky one given the different parameterizations in PyMC/Bambi/PreliZ we have 
(alpha, beta) and (mu, sigma) see here for details https://preliz.readthedocs.io/en/latest/examples/gallery/gamma.html

Usually, we use the mu, sigma parametrization like this

\begin{align}
y_i \sim \text{gamma}(\text{mu}=\mu_i, \text{sigma}=\frac{\mu_i}{\alpha^{0.5}}\) \\
\mu_i = \exp(\eta_i)\\
\eta_i = \beta_0 + \beta_1 \cdot x_i
\end{align}
 
 In brms, I think we have (mu, alpha) and Stan use (alpha, beta) [the same as the PyMC's default]

Reply:
Good point, there are clear typos here too. However the issue of different parameterizations is something we will run into in various places, also when recommending priors (e.g. exponential prior). I will open an issue about handling different parametrisations in general